pandas/tests/test_nanops.py

```diff
@@ -141,12 +141,12 @@ def _coerce_tds(targ, res):
             if axis != 0 and hasattr(
                     targ, 'shape') and targ.ndim and targ.shape != res.shape:
                 res = np.split(res, [targ.shape[0]], axis=0)[0]
-        except:
+        except (np.ValueError, IndexError):
```

Comment:
is np.ValueError different from ValueError?

Reply:
No, my mistake. `numpy.ValueError` does not exists. Changed it to `ValueError`.